api.md

```diff
@@ -943,6 +944,8 @@ to the server's timezone setting.
  with the ISO standard for Monday as the start of a week. This should only affect
```

Comment:
This does not only affect multi-day calls but all calls where 1 day divided by period has a remainder.

Reply:
Will address in a separate PR on the underlying text.